File: multiply_two_integers.py
def multiply_two_integers(a: int, b: int) -> int:
    return a * b


def main() -> None:
    first_number = int(input("Enter first integer: ").strip())
    second_number = int(input("Enter second integer: ").strip())
    print(f"Product: {multiply_two_integers(first_number, second_number)}")


if __name__ == "__main__":
    main()

File: MultiplyTwoIntegers.java
import java.util.Scanner;

public class MultiplyTwoIntegers {
    public static int multiplyTwoIntegers(int first, int second) {
        return first * second;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first integer: ");
        int firstNumber = scanner.nextInt();

        System.out.print("Enter second integer: ");
        int secondNumber = scanner.nextInt();

        System.out.println("Product = " + multiplyTwoIntegers(firstNumber, secondNumber));

        scanner.close();
    }
}
